.github/workflows/e2e_test_ci.yml

```diff
@@ -826,15 +793,7 @@ jobs:
 
       - name: Install spice
         run: |
-          chmod +x ./build/spice
-          chmod +x ./build/spiced
-          mkdir -p "$HOME/.spice/bin"
-          mv ./build/spice "$HOME/.spice/bin"
-          mv ./build/spiced "$HOME/.spice/bin"
-          echo "$HOME/.spice/bin" >> $GITHUB_PATH
-
-      - name: Check spice version
-        run: spice version
+          export SPICE_BUILD_ARTIFACTS_DIR=$PWD/build && ./test/scripts/install_spice.sh
```

Comment:
Couldn't we just make this an action like this?
https://github.com/spiceai/spiceai/blob/trunk/.github/actions/setup-rust/action.yml

Reply:
@digadeesh  -  yeah, @ewgenius  seems to also like Actions more than scripts where I prefer scripts as they are more reusable. As we have more people thinking Actions is a better choice I'll re-work this later today

Reply:
Updated